test/jdk/javax/swing/JTabbedPane/TestJTabbedPaneBackgroundColor.java

```diff
@@ -119,12 +126,14 @@ private static void createAndShowUI() {
         frame.setVisible(true);
     }
 
-    private static void doTesting(Point p, UIManager.LookAndFeelInfo laf) {
-        boolean isOpaque = pane.isOpaque();
+    private static void doTesting(Point p, UIManager.LookAndFeelInfo laf) throws Exception {
+        SwingUtilities.invokeAndWait(() -> {
+            isOpaque = pane.isOpaque();
+            c1 = pane.getBackground();
+            c2 = frame.getContentPane().getBackground();
+        });
         Color actual = robot.getPixelColor(p.x, p.y);
-        Color expected = isOpaque
-                ? pane.getBackground()
-                : frame.getContentPane().getBackground();
+        Color expected = isOpaque ? c1 : c2;
 
         if (!expected.equals(actual)) {
```

Comment:
You may want to print the color values to the test output.

Reply:
Updated.